### PR TITLE
Update create-signatures-and-disclaimers.md

### LINKED ISSUE #12954: information about inserting sender property variables is missing
information about inserting sender property variables is missing

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3a984a46-1f7b-6ef1-71f3-c5df6c142c01
* Version Independent ID: 4fab7da3-c9ae-0136-c693-ae99c1090f40
* Content: [Create organization-wide signatures and disclaimers - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/setup/create-signatures-and-disclaimers?view=o365-worldwide#create-a-signature-that-applies-to-all-messages)
* Content Source: [microsoft-365/admin/setup/create-signatures-and-disclaimers.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/setup/create-signatures-and-disclaimers.md)
* Service: **microsoft-365-business**
* GitHub Login: @twernermsft
* Microsoft Alias: **twerner**

### PR DESCRIPTION
**Pull request by msbemba**
Added sender property details to the rule 

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12954

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/create-signatures-and-disclaimers.md [hunk 1]
```diff
@@ -103,7 +103,9 @@ Go to the admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=850
 
 1. Choose **Stop processing more rules** if you only want this signature rule to run. 
 
-1. Choose **Defer the message** if rule processing doesn’t complete if you want this experience. 
+1. Choose **Defer the message** if rule processing doesn’t complete if you want this experience.
+
+1. Select ***Header** under the **Match sender address in message** drop down and click **Next** and **Finish**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/create-signatures-and-disclaimers.md]:
```suggestion
1. Select ***Header** under the **Match sender address in message** drop-down and then select **Next** and **Finish**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@twernermsft Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**